src/sp_execute.c

```diff
@@ -14,6 +13,10 @@ static int (*orig_zend_stream_open)(const char *filename,
 // FIXME handle symlink
 ZEND_COLD static inline void terminate_if_writable(const char *filename) {
   if (0 == access(filename, W_OK)) {
+    if (SNUFFLEUPAGUS_G(config).config_readonly_exec->dump)
```

Comment:
Care to run `clang-format --style="{BasedOnStyle: google, SortIncludes: false}"` on your code?